config/initializers/assets.rb

```diff
@@ -2,6 +2,12 @@
 
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
+Rails.application.config.before_configuration do
+  env_file = File.join(Rails.root, 'config', 'local_env.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.